README.md

```diff
@@ -617,5 +627,53 @@ From your AWS Management Console, perform the following steps:
     role, and copy the **Role ARN**. It should be in a format that looks like
     `arn:aws:iam::<your_aws_account_id>:role/JupiterOne`.
 
-[1]:
-  https://console.aws.amazon.com/iam/home#policies/arn:aws:iam::aws:policy/SecurityAudit
+## Events
+
+### Supported Events
+
+JupiterOne currently supports the following event sources and events:
+
+- S3
+  - CreateBucket
+  - DeleteBucket
```

Comment:
We should add UpdateBucket soon so that we capture the raw data of the event. Then we can update the updatedOn property and probably add an updatedBy from the principal.

Reply:
Yep, this list will be updated whenever I release full support for S3 events.